src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1108,7 +1109,12 @@ def set_up_service_bindings(self):
         if self.get_argument_service_bindings() is not None:
             linker_client = get_linker_client(self.cmd)
 
-            service_connectors_def_list, service_bindings_def_list = parse_service_bindings(self.cmd, self.get_argument_service_bindings(), self.get_argument_resource_group_name(), self.get_argument_name(), self.get_argument_customized_keys())
+            service_connectors_def_list, service_bindings_def_list = parse_service_bindings(self.cmd,
+                                                                                            self.get_argument_service_bindings(),
+                                                                                            self.get_argument_resource_group_name(),
+                                                                                            self.get_argument_name(),
+                                                                                            self.containerapp_def["properties"]["environmentId"],
```

Comment:
recommend to use safe_get

` safe_get(self.containerapp_def, "properties", 'environmentId')`

Reply:
done